<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyServer</class>
 <widget class="QMainWindow" name="MyServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DistributedBackup-SERVER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>641</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ServerLayout">
     <item>
      <widget class="QLabel" name="SAddressLabel">
       <property name="text">
        <string>Server address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SAddressLineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SPortLabel">
       <property name="text">
        <string>Server Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SPortLineEdit">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>641</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="DestinationLayout">
     <item>
      <widget class="QLabel" name="DAddressLabel">
       <property name="text">
        <string>Destination Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DAddressLineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DPortLabel">
       <property name="text">
        <string>Destination Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DPortLineEdit">
       <property name="text">
        <string>1234</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>641</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="FolderLayout">
     <item>
      <widget class="QLineEdit" name="FolderLineEdit">
       <property name="text">
        <string>/home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FindPushButton">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="StartButtonLayout">
     <item>
      <widget class="QPushButton" name="StartPushButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="TittleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>461</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DISTRIBUTED BACKUP &gt;&gt; SERVER PROGRAM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
